src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -52,9 +53,10 @@ const LargeScreenComponent = ({
       sx={{
         background: gradientStyle,
         borderRadius: '30px',
-        minHeight: '500px',
-        marginBottom: '100px',
-        flexDirection: 'row'
+        minHeight: '350px',
+        marginBottom: '100px'
+        // Below not necessary?
```

Comment:
Nope not necessary.  Default for `display: flex` is row already...

MUI `<Grid>` is based on flexbox

<img width="508" alt="Screenshot 2023-12-19 at 8 59 50 AM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/bfec99bb-2c6d-4fb3-a3b7-a2c110e18577">